org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -98,28 +96,54 @@
 				"--enable-xmlrpc",
 				"--enable-xmlwriter",
 				"--enable-xsl",
-				"--with-libzip",
 				"--enable-zlib",
-				"--with-zlib",
-				"--with-openssl"
+				"--with-argon2",
+				"--with-config-file-scan-dir=/usr/lib/sdk/php73/lib/ini:${HOME}/.config/php/7.3/ini",
+				"--with-curl",
+				"--with-gd",
+				"--with-libzip",
+				"--with-openssl",
+				"--with-tsrm-pthreads",
+				"--with-zlib"
 			],
 			"post-install": [
 				"install -Dm644 php.ini-development ${FLATPAK_DEST}/lib/php.ini"
 			],
 			"sources": [
 				{
 					"type": "archive",
-					"url": "https://www.php.net/distributions/php-7.3.5.tar.gz",
-					"sha256": "c953749b7f3310a3a74f920ef698f6d1c04636d11656ac9ffb3ab10d34e30e1e"
+					"url": "https://www.php.net/distributions/php-7.3.7.tar.gz",
+					"sha256": "4230bbc862df712b013369de94b131eddea1e5e946a8c5e286b82d441c313328"
+				}
+			]
+		},
+		{
+			"name":"xdebug",
+			"buildsystem":"simple",
```

Comment:
What's the reason to use `simple` buildsystem here, instead of the default, `autotools`?

Reply:
The need to run phpize before ./configure is run.

I've switched it to using autotools with a workaround, does that work better?

Reply:
You can run something before the build stage by adding a source node of type `shell`, e.g.
```json
{
	"type": "shell",
	"commands": [
		"${FLATPAK_DEST}/bin/phpize"
	]
}
```

Reply:
Changed.